### PR TITLE
Add a "Light Foreground" Color to the Themepack

### PR DESCRIPTION
**Pull request by thsparks**
This is useful for description text in the catalog:
image

Confirmed color contrast is okay against a white background using accessibility checker:
image

### CODE REVIEW

File: theme/themepacks.less [hunk 1]
```diff
@@ -37,6 +39,7 @@
 :root {
     /// Page
     --pxt-page-font: @pageFont;
+    --pxt-page-foreground-light: @pxt_page_foreground_light;
```
Comment (eanders-ms) [inline theme/themepacks.less:42]: Just checking.. will you be opening another PR to add this to the pxt repo?

Comment (thsparks) [reply theme/themepacks.less:42]: Yes, that's the plan. Just got delayed :)

**Review by eanders-ms — APPROVED** [review]

File: theme/themepacks.less (hunk 1 above)
Comment (thsparks) [reply theme/themepacks.less:42]: Here it is: https://github.com/microsoft/pxt/pull/9883
